static/js/user_status_ui.js

```diff
@@ -74,6 +75,7 @@ export function submit_new_status() {
         emoji_code: selected_emoji_info.emoji_code || "",
         reaction_type: selected_emoji_info.reaction_type || "",
         success() {
+            compose_actions.update_placeholder_text();
             close_user_status_modal();
```

Comment:
I think we need to do this from the `server_events_dispatch.js`  code path; this logic will only help if you're composing message to yourself.